Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -1,16 +1,36 @@
+using MLAgentsExamples;
 using UnityEngine;
 using Unity.MLAgents;
 using Unity.MLAgentsExamples;
 using Unity.MLAgents.Sensors;
+using BodyPart = Unity.MLAgentsExamples.BodyPart;
 
 public class WalkerAgent : Agent
 {
+    [Header("Walking Speed")]
+    [Space(10)]
     [Header("Specific to Walker")]
```

Comment:
You no longer need this line I think. "Specific to Walker" was to separate the base Agent's fields from the implemented Agent's fields.

Reply:
will fix